packages/react-charting/src/components/PieChart/Pie/Pie.tsx

```diff
@@ -1,50 +1,50 @@
 import * as React from 'react';
 import * as shape from 'd3-shape';
-import * as scale from 'd3-scale';
 import { IPieProps } from './Pie.types';
 import { LabeledArc } from '../Arc/Arc';
 import { IArcData } from '../Arc/Arc.types';
+import { FocusZone, FocusZoneDirection } from '@fluentui/react-focus';
 
-export class Pie extends React.Component<IPieProps, {}> {
+export class Pie extends React.Component<IPieProps, { focusedPie: string }> {
   public static defaultProps: Partial<IPieProps> = {
     pie: shape
       .pie()
+      .padAngle(0.01)
       .sort(null)
       /* eslint-disable @typescript-eslint/no-explicit-any */
       .value((d: any) => d.y),
   };
-  private colors: scale.ScaleOrdinal<string | number, any>;
```

Comment:
Previous Code
![image](https://github.com/microsoft/fluentui/assets/70366079/d4b379a8-d13f-415f-9723-9a921378b2b9)

Documentation Use  Case
![image](https://github.com/microsoft/fluentui/assets/70366079/cf41ac44-6cdf-4aac-8cca-ded2aa3c1a1f)


The behavior is same, just did some code refactoring here, the previous logic felt a little unneccessary, since we're using numerical indices as the scaleOrdinal, which we can directly do in the array

Current Code 
![image](https://github.com/microsoft/fluentui/assets/70366079/fe7dff04-d771-4e1f-a981-1706d841d9ee)

Lmk if I need to revert back to the previous way this was implemented

Reply:
noted, missed this one earlier

Reply:
Fixed it to match the previous color behavior with ordinalScale